src/types/types.ts

```diff
@@ -6,7 +7,9 @@ export interface Restaurant {
   location: string; // not sure
   rating: number;
   priceRange: number[];
+  mostPopular: boolean;
   keyId: string;
+  new: boolean;
```

Comment:
`new` is a bad word, many times it is a key word of the language, and will confuse other developers, or might even break your code

Reply:
@nitzanpap  done